pyproject.toml

```diff
@@ -1,7 +1,10 @@
+[tool.tox]
```

Comment:
Can you explain why this change is beneficial? Embedding an ini file as string in another ini file doesn't look like an improvement to me.

Reply:
`pyproject.toml` is a configuration file used by multiple projects, instead of having one configuration file for each. Particularly, apart from `tox`, I'd like to propose a PR for using `black`. Both `tox` and `black` support `pyproject.toml`. See, for instance, VUnit (https://github.com/VUnit/vunit/blob/master/pyproject.toml). See also https://github.com/carlosperate/awesome-pyproject.

Reply:
I would agree if tox would "natively" read pyproject.toml. But embedding ini within an ini is a step backwards IMO. I'd ask you to revert that. Having a different configuration file for a different tool isn't a problem in any way.

Reply:
Done.